packages/aio-sk-lib-telemetry/docs/guides/grafana.md

```diff
@@ -0,0 +1,420 @@
+
+The key steps are:
+
+1. **Start your local stack** using the same Docker Compose configuration from the Local Development section
+
+2. **Choose and start a tunnel** pointing to your OpenTelemetry Collector:
+   ```bash
+   # Example with Cloudflare Tunnel (Linux only)
+   docker run --rm -it --net=host cloudflare/cloudflared:latest tunnel --url http://localhost:4318
+
+   # Example with Cloudflare Tunnel (MacOS/Windows)
+   docker run --rm -it cloudflare/cloudflared:latest tunnel --url http://host.docker.internal:4318
+   ```
+
+3. **Note the tunnel URL** that gets generated (e.g. `https://abc123-def456-ghi789.trycloudflare.com`)
+
+#### Alternative: Add Tunneling to Docker Compose
+
+If you prefer to manage everything through Docker Compose, add this service to your existing `docker-compose.yml`:
+
+```yaml
+  # Add this service to your existing docker-compose.yml
+  cloudflared:
+    image: cloudflare/cloudflared:latest
+    container_name: cloudflared
+    restart: unless-stopped
+    networks: [telemetry]
+    depends_on: [otel-collector]
+    command: 
+      - "tunnel"
+      - "--url"
+      - "http://otel-collector:4318"
+```
+
+Then start the new container:
+```bash
+docker compose up
+```
+
+Check the `cloudflared` container logs to get your tunnel URL:
+```bash
+docker logs cloudflared
+```
+Look for a line like: `https://abc123-def456-ghi789.trycloudflare.com` - this is the URL you'll use in your telemetry configuration.
+
+### Updated Telemetry Configuration
+
+Replace the default localhost collector configuration with the tunnel URL in your telemetry setup:
+
+```ts
+// telemetry.ts
+import {
+  defineTelemetryConfig,
+  getAioRuntimeResource,
+  getPresetInstrumentations
+} from "@adobe/aio-sk-lib-telemetry";
+
+import {
+  OTLPTraceExporterProto,
+  OTLPLogExporterProto,
+  OTLPMetricExporterProto,
+  PeriodicExportingMetricReader,
+  SimpleLogRecordProcessor
+} from "@adobe/aio-sk-lib-telemetry/otel-api";
+
+function tunnelCollectorConfig(tunnelUrl: string) {
+  const makeExporterConfig = (path: string) => ({
+    url: `${tunnelUrl}/${path}`,
+  });
+
+  return {
+    traceExporter: new OTLPTraceExporterProto(makeExporterConfig("v1/traces")),
+    metricReader: new PeriodicExportingMetricReader({
+      exporter: new OTLPMetricExporterProto(makeExporterConfig("v1/metrics")),
+    }),
+    logRecordProcessors: [
+      new SimpleLogRecordProcessor(
+        new OTLPLogExporterProto(makeExporterConfig("v1/logs"))
+      ),
+    ],
+  };
+}
+
+export const telemetryConfig = defineTelemetryConfig((params, isDev) => {
+  // Use the tunnel URL instead of localhost
+  const tunnelUrl = "https://abc123-def456-ghi789.trycloudflare.com";
+
+  return {
+    sdkConfig: {
+      serviceName: "my-app-builder-app",
+      instrumentations: getPresetInstrumentations("simple"),
+      resource: getAioRuntimeResource(),
+      
+      ...tunnelCollectorConfig(tunnelUrl),
```

Comment:
When using the 
- deployed runtime actions, and
- production telemetry stack (not local)
Do we need to inform the `tunnelUrl` var for the config to work?
If that's the case, I would suggest renaming the var to `collectorUrl`, or `url`, since the use of a `tunnel` is specific to the use of local telemetry stack.